### PR TITLE
Fix - Filter stories by tag name

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

File: packages/core/src/content-delivery/ContentDelivery.ts [hunk 1]
```diff
@@ -264,7 +264,7 @@ export function createClient(
                     [`locale`]: localeCode ? { $in: [localeCode] } : undefined,
                     [`status`]: { $in: [Story.Status.PUBLISHED] },
                     [`visibility`]: { $in: [Story.Visibility.PUBLIC] },
-                    [`tag_names`]: tags?.length ? { $in: [tags] } : undefined,
+                    [`tag.name`]: tags?.length ? { $any: [tags] } : undefined,
```
Comment (yuriyyakym) [inline packages/core/src/content-delivery/ContentDelivery.ts:267]: What's the difference between `any` and `in` in case of search predicate?

Comment (yuriyyakym) [reply packages/core/src/content-delivery/ContentDelivery.ts:267]: Would `tags?.length ? { $in: tags } : undefined` also work?

**Review by yuriyyakym — APPROVED** [review]

**Review by mohammadxali — APPROVED** [review]

File: packages/core/src/content-delivery/ContentDelivery.ts (hunk 1 above)
Comment (fgyimah) [reply packages/core/src/content-delivery/ContentDelivery.ts:267]: This particular filter only supports `$any`, `$some` and `$none` filters, $in is not supported

Comment (yuriyyakym) [reply packages/core/src/content-delivery/ContentDelivery.ts:267]: But is there any difference between those predicates? 🤔

Comment (fgyimah) [reply packages/core/src/content-delivery/ContentDelivery.ts:267]: I'm not sure there is

**Pull request merged**